packages/js-example-nodejs/src/index.cjs

```
@@ -0,0 +1,108 @@
+    // we are sure that we have an SVG node here, it is always created at Graph creation
+    const svgNode = this.graph.container.querySelector('svg');
+    return `${xmlFileHeader}
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+     width="${bounds.width}" height="${bounds.height}" viewBox="${bounds.x} ${bounds.y} ${bounds.width} ${bounds.height}">
+${svgNode.innerHTML}
+</svg>`;
+  }
+}
+
+console.info('Exporting SVG...');
```

ESLint

[error] Unexpected console statement.

(no-console)